app/controllers/concerns/foreman/controller/ip_from_request_env.rb

```diff
@@ -0,0 +1,18 @@
+module Foreman
+  module Controller
+    module IpFromRequestEnv
```

Comment:
I've been thinking about using https://api.rubyonrails.org/classes/ActionDispatch/RemoteIp.html so we can rely on the standard request IP mechanism but never really knew how to get started. Probably out of scope for this refactor but I'd like your thoughts on this. It is relevant when we do pick up https://github.com/theforeman/puppet-foreman/pull/677 again.

Also note that then setups might have two IPs in `HTTP_X_FORWARDED_FOR` which the RemoteIp middleware can correctly parse.

Reply:
I think we should use rails standards if we can, but I concur: This is out of the scope of this PR.